WcaOnRails/config/locales/pt.yml

```diff
@@ -3584,7 +3613,7 @@ pt:
       title: Política de proximidade
       #original_hash: 79ed06e
       content1: >-
-        Competições da WCA serão aceites se acontecerem com pelo menos 100 km
+        Competições da WCA serão aceites se acontecerem com pelo menos 100 km 
```

Comment:
This line adds some trailing whitespace, which caused the build to fail.

@jonatanklosko, was this whitespace generated by the internationalize app when rendering to yaml?

Reply:
@jfly I've just checked the database and as suspected there's just double space after `km`. YAML `>-` let's you split string into multiple lines by treating `\n` as space, that's why there's exactly one trailing space.

Reply:
Got it. So in that case, I guess we just have to ask António to fix this and resubmit?

Reply:
Yeah, already did that =)